tests/integration/cli/projects/script/scripts/_utils.py

```diff
@@ -0,0 +1,2 @@
+# Note that any file prepended w/ underscore is ignored
+deploy = lambda: None  # noqa: E731
```

Comment:
```suggestion
def deploy():
    pass
```

This is not an anonymous method and there is no reason to use lambda then